### DISCUSSION TITLE
[Thread] Integration Problems/Questions for ML service

### DISCUSSION THREAD

**Discussion by produdez** [dirtyg2120/thesis-backend#45] [Machine learning]
Let us summarize all our `hickups` with ML model in this doc so we can discuss and work on it later after our meeting

[Doc Link](https://docs.google.com/document/d/1yiNOV9eSZu7yMtK5OloNi1i3iDb8A9fRgotbmAEvx4k/edit?usp=sharing)

@phhai-hcmut @dirtyg2120 @produdez @dfighter1312

**Comment by phhai-hcmut** [comment]
I think Discussion is more suitable than Issue for this.
@dirtyg2120 https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/enabling-or-disabling-github-discussions-for-a-repository

**Reply by produdez** [reply]
wao dint even know there's discussion feature :V nice
